android/src/main/java/com/instabug/reactlibrary/RNInstabugReactnativeModule.java

```diff
@@ -1056,6 +1056,17 @@ public void run() {
         });
     }
 
+    @ReactMethod
+    public void willRedirectToStore() {
+        MainThreadHandler.runOnMainThread(() -> {
+            try {
+                Instabug.willRedirectToStore();
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        });
```

Comment:
We can't use lambda functions since they are supported starting from Java 8 and we still haven't dropped support for previous versions, so for compatibility let's change the `() -> { ... }` lambda syntax to an anonymous `Runnable` class as used in the other methods.